rdt/transformers/datetime.py

```diff
@@ -156,7 +158,10 @@ def _reverse_transform(self, data):
             datetime_data = pd.Series(datetime_data)
 
         if self.datetime_format:
-            datetime_data = datetime_data.dt.strftime(self.datetime_format)
+            if self._dtype == 'object':
+                datetime_data = datetime_data.dt.strftime(self.datetime_format)
+            elif 'datetime' in self._dtype.name:
```

Comment:
is there a way to check that a dtype is a datetime dtype other than checking the string?

Reply:
Yes we can use `is_datetime64_dtype`  from `pandas` api. 👍🏻